tests/test_iam/test_iam.py

```diff
@@ -1690,11 +1690,16 @@ def test_get_account_authorization_details():
     assert result["RoleDetailList"][0]["AttachedManagedPolicies"][0][
         "PolicyArn"
     ] == "arn:aws:iam::{}:policy/testPolicy".format(ACCOUNT_ID)
+    print(result["RoleDetailList"][0]["RolePolicyList"][0]["PolicyDocument"])
```

Comment:
Remove both print-statements please - happy to merge after that

Reply:
Oops... removed